### PR TITLE
Large scale change: pre-commit checks vol2

### PR DESCRIPTION
**Pull request by kaidokert**
Runs `pre-commit --all` and fixes formatting in a number of low-churn, low risk files. This is according to current linter+formatter standards.

b/278726440

### CODE REVIEW

**Comment by kaidokert** [comment]
First PR of this series in #1010

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -122,8 +122,12 @@ repos:
         exclude: |
             (?x)^(
                 .pre-commit-config.yaml$|
+                cobalt/base/unicode/|
+                cobalt/math/|
```
Comment (hlwarriner) [inline .pre-commit-config.yaml:126]: Why should this be excluded?

**Comment by hlwarriner** [comment]
Can you please update the bug reference in the commit message? It looks like the wrong one.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (kaidokert) [reply .pre-commit-config.yaml:126]: Because this is not our code, it's all copyright Chrome and forked from there.

**Comment by kaidokert** [comment]
> Can you please update the bug reference in the commit message? It looks like the wrong one.

Thanks for catching this, updated the PR description - that's where the landed commit gets the bug ID now. I had dropped a zero ( b/278726440 is the correct one, or alternatively b/199280707 )

**Pull request merged**